audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/stream/config/AkkaComponentsConfig.java

```diff
@@ -64,11 +80,12 @@
     }
 
     @Bean
-    Source<CommittableMessage<String, AuditSuccessMessage>, Control> successCommittableRequestSource(final ActorSystem actorSystem, final ConsumerTopicConfiguration configuration) {
+    Source<CommittableMessage<String, AuditSuccessMessage>, Control> successCommittableRequestSource(final ActorSystem actorSystem,
+                                                                                                     final ConsumerTopicConfiguration configuration) {
         ConsumerSettings<String, AuditSuccessMessage> consumerSettings = SUCCESS_INPUT_COMPONENTS.consumerSettings(
                 actorSystem,
                 SerDesConfig.successKeyDeserializer(),
-                SerDesConfig.successValueDeserializer());
+                SerDesConfig.successValueDeserializer(configProperties));
 
         Topic successTopic = configuration.getTopics().get("success-topic");
```

Comment:
Makes sense to add `getTopic(String)` to `ConsumerTopicConfiguration`

Reply:
This would different to how the other services are doing it, would it not be best to have some sort of consistency throughout the services? I am happy to add it in though.

Reply:
I've seen lots of code doing this. Better to ask the object direct for a single topic imo. I usually then add `Optional<Topic> findTopic(String)`. This makes using the api easier. You can then make `getTopic(String)` throw an exception if it's not found. This removes the burden of having to test for null everywhere, resulting in less code.

Reply:
Okay, I will add this in.